### DISCUSSION TITLE
☕ Make software packaging... a cup of tea.xyz :fireworks:

### DISCUSSION THREAD

**Discussion by adriens** [pkgxdev/pkgx#550] [Ideas]
[☕ Make software packaging... a cup of tea.xyz ](https://dev.to/adriens/make-software-packaging-a-cup-of-teaxyz-3h77)

![image](https://user-images.githubusercontent.com/5235127/234122533-092f3360-45c8-41af-9700-df85c49e7120.png)

**Comment by adriens** [comment]
:bulb: I would have rather posted this to a "Show & tell discussion" category.

**Reply by mxcl** [reply]
@mfts show n' tell is probs worth having right?

**Reply by adriens** [reply]
BTW :point_right: I have some internal feedback i nmy team, asking for more about tea scripts and the sandbox scenario

**Reply by mfts** [reply]
@adriens "Show and Tell" category is now available! Amazing work.
